d365fo.tools/tests/general/Help.Example.Tests.ps1

```diff
@@ -42,7 +42,7 @@ foreach ( $commandName in $commands) {
 
             if ( ($example -like "*|*" ) -or (-not ($example -match $commandName)) -or ($example -like "*).*")) {
                 It "Example - $example" -Skip { $true }
-            } elseif ($example -like "*=*") {
+            } elseif ($example -match '(?<=^(([^"]\*(?<!\\)"[^"]\*(?<!\\)"[^"]\*)\*|[^"]*))=') {
```

Comment:
That is some crazy regex !

Are we currently only supporting double quotes? How much would it take make it skip single quotes also?

Below syntax is valid and should yield the same result from a PowerShell perspective...

```powershell
Invoke-D365SqlScript -Command "DELETE FROM SALESTABLE WHERE RECID = 123456789"
Invoke-D365SqlScript -Command 'DELETE FROM SALESTABLE WHERE RECID = 123456789'
```

Reply:
This might work - based on some super quick mocking around:

```
'(?<=^(([^"|^'']\*(?<!\\)"[^"|^'']\*(?<!\\)"[^"|^'']\*)\*|[^"|^'']*))='
```

Reply:
I will check on supporting single quotes as well. Also, I didn't come up with the regex. 😁 I got it from here: https://robrich.org/archive/2007/11/29/regex-match-content-unless-it-is-inside-quotes.aspx

Reply:
You are being to modest. You got better google-fu than me and that earns some street credits around here!

Reply:
Even reading the blog post, doesn't help me to fully grasp what it does.

But - it seems to work, and that is what we are after! 🤘